helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -281,7 +281,7 @@ or modify your existing configuration to migrate to a new Auth0 Tenant.
 
 ### From Before FiftyOne Teams Version 1.1.0
 
-The FiftyOne 0.15.0 SDK (database version 0.23.0) is _NOT_ backwards-compatible
+The FiftyOne 0.15.1 SDK (database version 0.23.1) is _NOT_ backwards-compatible
```

Comment:
I don't see changes to `helm/fiftyone-teams-app/README.md`? 

Maybe the the pre-commit hooks weren't run to render the changes in the template `helm/fiftyone-teams-app/README.md.gotmpl` to `helm/fiftyone-teams-app/README.md` ?

Reply:
Fixed. I ran pre-commit, commited the changes and pushed.